### ISSUE TITLE
some code issues

### ISSUE THREAD

**Issue by zhendebuxiangxuexihaoma** [akashsdoshi96/obq-gan-mimo-ce#1]
I would like to ask why the downloaded .ipynb file cannot be run in the jupyter notebook. The error displayed is that some variables in it are not defined, such as CDL. How to solve this situation?

**Comment by akashsdoshi96** [comment]
You need to download all the channel data into the /data folder. The necessary instructions can be found at https://github.com/akashsdoshi96/ota-gan-mimo-ce/blob/main/data/README.md

**Comment by zhendebuxiangxuexihaoma** [comment]
I would like to ask why there are some undefined variables in the Jupyter Notebook when using test files. I have searched through the code and found no statements defining this variable.
![6XTI would like to ask why there are some undefined variables in the Jupyter Notebook when using test files. I have searched through the code and found no statements defining this variable.
~DY}$3VLRU{~~VK1KH](https://github.com/akashsdoshi96/obq-gan-mimo-ce/assets/134215504/f0b801eb-2add-4efe-91d3-081cdac72dbf)
